docs/source/Notify/N001.rst

```diff
@@ -0,0 +1,195 @@
+  Services such as SMTP2go will act as an SSL bridge from a non-secure sender.
+  Personal (non-commercial) accounts are available at no cost too.
+
+The table below provides sample entries for smtp2go.com and smtp.smart-mail.de
+
++-----------+--------------------+--------------------+--------------------------------------+
+| Name      |     Example 1      |     Example 2      |          Description                 |
++===========+====================+====================+======================================+
+| Domain:   |    smtp2go.com     |   smart-mail.de    |       Mail provider domain.          |
++-----------+--------------------+--------------------+--------------------------------------+
+| Server:   |  mail.smtp2go.com  | smtp.smart-mail.de |        Mail server address.          |
++-----------+--------------------+--------------------+--------------------------------------+
+| Port:     |        2525        |         25         |  SMTP port recommended by provider.  |
++-----------+--------------------+--------------------+--------------------------------------+
+| Sender:   |   joe@gmail.com    | joe@anydomain.com  |   Sender's email address.            |
++-----------+--------------------+--------------------+--------------------------------------+
+| Receiver: |   Jim@yahoo.com    |  Jim@domain.com    |   Recipient's email address.         |
+|           |                    |                    |   Note: Multiple allowed,            |
+|           |                    |                    |   use comma (,) between each address.|
++-----------+--------------------+--------------------+--------------------------------------+
+| Subject:  |    Subject Text    |    Subject Text    |  Topic in email headline.            |
++-----------+--------------------+--------------------+--------------------------------------+
+| User:     |      ESP123        | ESP@smart-mail.de  |  Username to the SMTP Server.        |
++-----------+--------------------+--------------------+--------------------------------------+
+| Pass:     |   secretepw001%%   |   secretepw002$$   |  Password to the SMTP Server.        |
++-----------+--------------------+--------------------+--------------------------------------+
+| Body:     |    Device uptime is %uptime% mins.<br>  |  512 characters max.                 |
+|           |    Connected via WiFi %ssid%.           |  Use <br> for new line.              |
++-----------+--------------------+--------------------+--------------------------------------+
+| Enabled:  |     Check Box      |     Check Box      |  Email is disabled if box unchecked. |
++-----------+--------------------+--------------------+--------------------------------------+
+
+
+:raw-html:`<br clear="all">`
+
+.. _adv_feat:
+
+Advanced Features
+------------------
+
+Along with the pre-defined settings, it is possible to customized the email message body and recipient address from a Notify command rule.
```

Comment:
You're not mentioning the feature of adding a custom subject as the 3rd argument. Can you add that too?
There's no special processing on the subject, other than it replacing the configured subject.

Reply:
Thanks for finding that. I will add it to the RTD.